tools/m2-repo-cleaner/src/main/java/org/fusesource/ide/tools/mavenutils/M2RepoCleaner.java

```diff
@@ -101,4 +118,12 @@ private static void handleCorruptedZip(File file) {
 			}
 		}
 	}
+	
+	private static void handleLastUpdatedFile(File file) {
+		handleCorruptedZip(file);
+//		corruptedFiles.add(file.getPath());
```

Comment:
I still have to check that but in the last couple of CI runs the number of deleted/affected files was not that it included the whole m2 repo. Latest run says:
"FINISHED - We checked 5303 files in 15595 folders with 470 corrupted files detected!"